_dev/apps/ui/stories/merchant-center-account-card.stories.ts

```diff
@@ -134,6 +134,18 @@ EnabledButNoAccount.args = {
   }
 };
 
+export const EnabledWithOneAccountFromModule:any = Template.bind({});
+EnabledWithOneAccountFromModule.args = {
+  isEnabled: true,
+  loading: false,
+  initialMcaStatus: Object.assign({},
+    merchantCenterAccountNotConnected,
+    {
+      aggregatorId: '123456'
```

Comment:
Once the other comments are taken in account:

```suggestion
      aggregatorId: '381504106'
```